chore: update docs for algo and hbar recoveries (#408)
BitGo/wallet-recovery-wizard: BROADCAST_TRANSACTION.md

# Broadcast Transaction Instructions

> WRW now supports natively broadcasting transactions from the UI itself. User needs to select the coin and copy the fully signed serialized transaction. For users that use OVC (Offline Vault Console), this is the 'txHex' field from the fully signed JSON file.

Reviewer: ```suggestion
WRW natively supports broadcasting transactions from the UI. To do so, select the coin and copy the fully-signed serialized transaction. If using the Offline Vault Console (OVC), use the 'txHex' field from the-fully signed JSON file.
```

### This feature is currently supported for the following coins -
- HBAR

> - ALGO (Note: BitGo doesn't have an external node for Algo and requires clients to input the details of their node)

Reviewer 2: This should probably say that there is no public node that can be used.

> - ALGO (Note: BitGo doesn't have an external node for Algo and requires clients to input the details of their node)

Reviewer: ```suggestion
- ALGO (Note: BitGo doesn't have an external node for Algo, so you must input your own node the details)
```
